core/controllers/admin_test.py

```diff
@@ -200,6 +681,86 @@ def test_invalid_username_in_view_and_update_role(self):
             csrf_token=csrf_token,
             expected_status_int=400)
 
+    def test_cannot_view_role_with_invalid_view_method(self):
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+        response = self.get_json(
+            feconf.ADMIN_ROLE_HANDLER_URL,
+            params={'method': 'invalid_method', 'username': 'user1'},
+            expected_status_int=400)
+
+        self.assertEqual(response['error'], 'Invalid method to view roles.')
+
+    def test_deassign_user_from_all_topics(self):
```

Comment:
This seems more like assigning a different role than deassigning a user ... ?

Reply:
done